lib/src/scss-shared/_nx-text-helpers.scss

```diff
@@ -31,6 +31,24 @@
   white-space: nowrap;
 }
 
+// Should be called on a custom class attached to the <body> tag
```

Comment:
I'm not sure I follow what this means by "should be called on a custom class"

Reply:
> I'm not sure I follow what this means by "should be called on a custom class"

Improved the comment here: https://github.com/sonatype/sonatype-react-shared-components/pull/89/commits/e7519e4dce13ee2b2ef0373a0c8edeb30c8bdedc